regression-tests/backends/gmysql-master

```diff
@@ -23,9 +23,11 @@ gmysql-dbname=$GMYSQLDB
 gmysql-user=$GMYSQLUSER
 gmysql-host=$GMYSQLHOST
 gmysql-password=$GMYSQLPASSWD
+
+any-to-tcp=no
 __EOF__
 
-		gsql-master gmysql
+		gsql-master gmysql dyndns
```

Comment:
Ah, like that. I always get confused about skipreasons. So we test mysql(+dyndns) with any-to-tcp off, and skip the actual any-to-tcp test; then for all other backends, we don't test dyndns (we never did) and we do test any-to-tcp. Sounds good.